documentation/book/cluster-operator.adoc

```diff
@@ -703,24 +703,33 @@ Note line 12, where the the `strimzi-cluster-operator` ServiceAccount is specifi
 
 ==== Defining a Role
 
-The operator needs to operate using a Role that gives it access to the necessary resources
+The Cluster Operator needs to operate using a Role that gives it access to the necessary resources.
+Depending on {ProductPlatformName} cluster setup, a cluster administrator might be needed to create the Role.
+Cluster administrator rights are only needed for the creation of the Role.
+The Role it self doesn't contain any cluster administrator privileges and the Cluster Operator will not run under cluster admin account.
+The Role follows the principle of least privilege and contains only the privileges needed by the Cluster Operator to operate Kafka, Kafka Connect and Zookeeper clusters.
+The assigned privileges allow Cluster Operator to manage {ProductPlatformName} resources such as StatefulSets, Deployments, Pods and ConfigMaps.
 
 [source,yaml,options="nowrap"]
 .Example Role for the Cluster Operator
 ----
 include::examples/install/cluster-operator/02-role.yaml[]
 ----
 
+When the Role cannot be created, the predefined ClusterRole named `edit` can be used instead.
+
 ==== Defining a RoleBinding
 
-Finally, the operator needs a RoleBinding which associates its Role with its ServiceAccount
+Finally, the operator needs a RoleBinding which associates its Role with its ServiceAccount:
 
 [source,yaml,options="nowrap"]
 .Example RoleBinding for the Cluster Operator
 ----
 include::examples/install/cluster-operator/03-role-binding.yaml[]
 ----
 
+In case the `edit` ClusterRole is being used, the RoleBinding has to be modified to refer to it instead of the `strimzi-cluster-operator-role` Role.
+
```

Comment:
do we want to be explicit and showing how/where with an YAML snippet ?

Reply:
Yes, had the same thought: We should elaborate on how to do this.

Reply:
To be honest, I do not think we should go into too much detail here. The `edit` role will not be a fully sufficient after Paolo's PR for `broker.rack` support. And offering YAML for using it might encourage people too much to use it instead of creating the proper roles.

Reply:
Ok, I read the missing reactions as that you don't agree with me. So I added the example :-o.

Reply:
No @scholzj my missing reaction was due to working on the rack feature then lunch then children at school :-) I see your point, we are adding an example that won't work in the future.

Reply:
Well, too late, it is there now. It will be anyway only alternative even after your update. We will just need to add a note that they have to disable the racks.